src/mailroom.py

```diff
@@ -0,0 +1,93 @@
+"""Automate thankyou notes and generates donor reports."""
+
+import io
+import sys
+from sys import version_info
```

Comment:
Looks like some duplicated imports here. It won't effect the functionality, but make sure to clean this kind of thing up.